Unreal/SystemLocalized/det/UnrealShare.det

```diff
@@ -1529,15 +1529,15 @@ FailtureMessage="Sie benötigen einen Schlüssel, um diese Tür zu öffnen."
 
```

Comment:
Line 1528: "Sie benötigen einen Schlüssel, um diese Tür zu öffnen." → "Du benötigst einen Schlüssel, um diese Tür zu öffnen." for consistency reasons.